dragonfly/session/handler_player_action.go

```diff
@@ -55,11 +56,26 @@ func handlePlayerAction(action int32, face int32, pos protocol.BlockPos, entityR
 		s.c.AbortBreaking()
 	case protocol.PlayerActionStopBreak:
 		s.c.FinishBreaking()
+	case protocol.PlayerActionContinueDestroyBlock:
+		if s.c.Breaking() {
+			s.c.AbortBreaking()
+			s.c.StartBreaking(world.BlockPos{int(pos[0]), int(pos[1]), int(pos[2])}, world.Face(face))
+		}
```

Comment:
This also looks a bit... off, are you sure this isn't just supposed to do the exact same thing as the case below?

Reply:
Changed